plugins/org.eclipse.n4js/src/org/eclipse/n4js/scoping/utils/ProjectImportEnablingScope.java

```diff
@@ -86,9 +82,9 @@
 	 * such cases; however, project imports will not be available in such tests.
 	 */
 	public static IScope create(IN4JSCore n4jsCore, ImportDeclaration importDecl, IScope parent, IScope delegate) {
-		if (n4jsCore == null || importDecl == null || parent == null) {
-			throw new IllegalArgumentException("none of the arguments may be null");
-		}
+		// if (n4jsCore == null || importDecl == null || parent == null) {
```

Comment:
Why is check commented out?

Reply:
hmm... looks like by mistake. I will run builds with check in place